src/locale/properties/shared/shared.en.properties

```diff
@@ -124,4 +126,12 @@ shared.link=Link
 shared.linkDescription=A link to a profile page or description of the role. Links should be in the full URL format e.g. http://www.website.com/page.html
 shared.invalidLink=Invalid Link
 shared.visibilityDescription=Manage who can see this information in your record.
+shared.actions=Actions
+shared.combineWorks=Combine works
```

Comment:
seems like `shared.combineWorks` is not use

Reply:
it is been used here https://github.com/ORCID/orcid-angular/blob/926d77fc187e278ad1ea5864b7c68a8510700c64/src/app/record/components/work-stack-group/work-stack-group.component.html#L61